src/store/getters.ts

```diff
@@ -991,7 +1005,7 @@ export const getDraftNameReservation = (state: StateIF): DraftReqI => {
         }
         if (!data['additionalInfo'].includes('*** Legal Name:')) {
           data['additionalInfo'] += '\n\n'
-          let notice = `*** Legal Name: ${this.nameChoices.name1} ***`
+          let notice = `*** Legal Name: ${getNameChoices(state)?.name1} ***`
```

Comment:
Cam, this is correct, right?

Reply:
Syntax wise? Yes, i think so when using a getter inside another getter. Does this give you any lint errors or is there a type on the return from the getter?

Reply:
No, the change from `this.nameChoices` (which showed an error due to "this") to the local getter (which I sort of copied from another method in here).

Reply:
Ie, I fixed an error, but it feels weird to call a getter like this :P

Reply:
Referring the lack of `this`?

Reply:
Yes.

Reply:
For sure, i'm not sure the terminology but the getters don't belong to a class so aren't scoped? Is that the reason maybe?

Reply:
I think so. I think this is correct but I wanted to let you know.